content/guides/models/track/runs/multiple-runs-per-process.md

```diff
@@ -0,0 +1,168 @@
+---
+description: Manage multiple runs in a single Python process using W&B’s `reinit` functionality
+menu:
+  default:
+    identifier: multiple-runs
+    parent: what-are-runs
+title: Create and manage multiple runs in a single process
+---
+
+Manage multiple runs in a single Python process. This is useful for workflows where you want to keep a primary process active while creating short-lived secondary processes for sub-tasks. Some use cases include:
+
+- Keeping a single “primary” run active throughout a script while spinning up short-lived “secondary” runs for evaluations or sub-tasks.  
+- Orchestrating sub-experiments in a single file.  
+- Logging from one “main” process to several runs that represent different tasks or time periods.
+
+By default, W&B assumes each Python process has only one active run at a time when you call `wandb.init()`. If you call `wandb.init()` again, W&B will either return the same run or finish the old run before starting a new one—depending on the configuration. The content in this guide explains how to use `reinit` to modify the `wandb.init()` behavior to enable multiple runs in a single Python process.
+
+
+## `reinit` options
+
+Use the `reinit` parameter to determine how W&B handles multiple calls to `wandb.init()`. `reinit` accepts:
+
+- `create_new`: Create a new run (`wandb.init()`) without finishing existing, active runs. 
```

Comment:
This would be easier to read as a table, I think, or as H3 headings where you can give the specifics about each one. There is some beefy stuff here that might get missed in the current presentation.